com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/TextureSetGenerator.java

```diff
@@ -286,17 +299,15 @@ private static SpriteGeometry.Builder createPolygonUVs(SpriteGeometry geometry,
 
         return geometryBuilder;
     }
-    /**
-     * Generate an atlas for individual images and animations. The basic steps of the algorithm are:
-     *
-     * 1. Extrude image borders
-     * 2. Layout images
-     * 3. Shrink inner rects by previous extrusion
-     * 4. Create vertex data for each frame (image) in each animation
+
+    /* Calculate the layout of a set of images
+     * 1. Extrude rect borders and add inner padding
+     * 2. Layout rects
+     * 3. Shrink rects by previous extrusion
      */
-    public static TextureSetResult calculateLayout(List<Rect> images, List<SpriteGeometry> imageHulls, int use_geometries,
-                AnimIterator iterator, int margin, int innerPadding, int extrudeBorders,
-               boolean rotate, boolean useTileGrid, Grid gridSize, float maxPageSizeW, float maxPageSizeH) {
+    public static LayoutResult calculateLayoutResult(List<Rect> images, int margin, int innerPadding, int extrudeBorders, boolean rotate,
+                                                    boolean useTileGrid, Grid gridSize, float maxPageSizeW, float maxPageSizeH) {
+        TimeProfiler.start("calculateLayoutResult");
```

Comment:
We'll be calling this from multiple threads when building in the editor. Will that mess up the profiler?